pkg/envoy/rds/route/route_config.go

```diff
@@ -287,6 +289,39 @@ func buildWeightedCluster(weightedClusters mapset.Set) *xds_route.WeightedCluste
 	return &wc
 }
 
+// TODO: Add validation webhook for retry policy
```

Comment:
Creating an issue for this would be helpful to track all the work remaining in retry

Reply:
Just add this as a task in the existing issue. No need to create multiple issues as the primary implementation task is still open.

Reply:
that works, we just need some form of tracking to know the work remaining. This can be done on the parent issue the change is a part of

Reply:
Thanks @shashankram for adding TODOs to the parent issue!